awpy/visualization/plot.py

```diff
@@ -565,19 +562,17 @@ def create_round_map_control_gif(
     if map_name not in NAV:
         msg = "Map not found."
         raise ValueError(msg)
-    if "tmp" not in os.listdir():
-        error_string = (
-            "Frame visualizations cannot be saved to the"
-            "./tmp folder - it does not exist"
-        )
-        raise ValueError(error_string)
+
+    if os.path.isdir(_TMP_FOLDER):
+        shutil.rmtree(f"{_TMP_FOLDER}/")
+    os.mkdir(_TMP_FOLDER)
 
     images: list[np.ndarray] = []
     print("Saving/loading frames")
     frames = round_data["frames"]
-    i = 0
-    for frame in frames or []:
-        filename = f"./tmp/tmp{i}.png"
+
+    for i, frame in enumerate(frames) or []:
```

Comment:
The pyright issue is that the enumerate here has to include the `or []`

Reply:
Ah thank you makes sense. Addressed in next commit to branch.